src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -0,0 +1,133 @@
+package com.wikia.webdriver.testcases.mercurytests.curatedcontenttests;
+
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.contentpatterns.MercuryMessages;
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.CuratedMainPagePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.CuratedContentPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.MercuryBasePageObject;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class NavigationTests extends NewTestTemplate {
+
+  @BeforeMethod(alwaysRun = true)
+  public void prepareTest() {
+    driver.manage().timeouts().pageLoadTimeout(100, TimeUnit.SECONDS);
+  }
+
+  private static final String ROOT_PATH = "/";
+  private static final String ROOT_PATH_SECTION = "/main/section/";
+  private static final String ROOT_PATH_CATEGORY = "/main/category/";
+
+  private enum PageElements {
+    SECTION_TITLE("Section title"),
+    LINK_TO_MAIN_PAGE("Link to main page"),
+    SECTION("Section as the container of many elements"),
+    SECTION_ITEM("Item in a section");
+
+    private String name;
+
+    private PageElements(String name) {
+      this.name = name;
+    }
+  }
+
+  // CCT06
+  @Test(groups = {"MercuryNavigationTests_001", "MercuryNavigationTests", "Mercury"})
+  public void MercuryNavigationTests_001_navigateThroughCategory() {
+    wikiURL = urlBuilder.getUrlForWiki(MercuryWikis.MERCURY_CC);
+    boolean result;
+
+    MercuryBasePageObject base = new MercuryBasePageObject(driver);
+    CuratedMainPagePageObject mainPage;
+    mainPage = base.openCuratedMainPage(wikiURL, MercurySubpages.CC_MAIN_PAGE);
+    CuratedContentPageObject category = mainPage.tapOnCuratedElement(2);
```

Comment:
wouldn't it be better to instead of tapping by index you tap by label? It is less probable label is going to change and it's easier to figure out what is actually tapped on the screen.